plugin.yml

```diff
@@ -65,6 +65,8 @@ configuration:
       type: boolean
     sysctls:
       type: array
+    storage-opt:
+      type: string
```

Comment:
Makes sense! 👌
We probably should order attributes in alphabetical order ... but that's outside this PR